content/tutorials/coordination.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Coordination Planner"
+date: 2021-04-25T15:14:54+10:00
+featured: true
+draft: false
+duration: 30
+active: true
+difficulty: Intermediate
+summary: Initialize and operate multiple MuSHRs car to coordinationally complete tasks  
+weight: 3
+---
+
+<h2> By: <a href='https://www.linkedin.com/in/p-chanrun/'>Podshara Chanrungmaneekul</a> </h2>
+<img src="/tutorials/coordination_planner/header.gif" width="800" >
+
+## Introduction
+
+### Goal 
+
+This tutorial will teach you how to setup the Enhance Conflict-Based Search with Optimal Task Assignment (ECBS-TA) planning algorithm on a set of MuSHR cars. ECBS-TA produces a set of collision free trajectories, one for each car, from start to goal. By the end of the tutorial, a set of cars will be able to generate path that avoiding each other and complete the given set of tasks.
+
+### Requirements
+
+- If in sim, complete the [quickstart tutorial](https://mushr.io/tutorials/quickstart/)
+- If on real car, complete the [first steps tutorial](https://mushr.io/tutorials/first_steps/)
+- Navigation System [mushr_navigation_system](https://github.com/naughtyStark/nhttc_ros)
+
+## Coordination Planner Problem Statement
+
+There is a team of `num_agents` MuSHR cars and a set of `num_goals` tasks. Each task requires one car to follow all of the `num_waypoints` points and stop at the last one. The environment is a grid world where the movement of the car is restricted to orthongonal movement. This approximate approach can work well when combined with the [nhttc controller](https://github.com/naughtyStark/nhttc_ros). It is possible that some cars will not have any tasks and others will have multiple to complete sequentially.
+
+## Install
+Clone repo and install the package: 
+{{< highlight bash >}}
+$ cd ~/catkin_ws/src/
+$ git clone https://github.com/prl-mushr/mushr_coordination.git
+$ cd mushr_coordination
+$ git clone https://github.com/whoenig/libMultiRobotPlanning.git
+$ cd ~/catkin_ws/
+$ catkin_make
```

Comment:
I'm having lots of build errors

Reply:
What is the build error? I ran it fresh multiple times and nothing was wrong.